### DISCUSSION TITLE
maiko repository doesn't have "Issues" section

### DISCUSSION THREAD

**Discussion by MattHeffron** [Interlisp/medley#1463] [GitHub use]
The maiko repository (and some others) doesn't have an "Issues" section.
The medley, clhs, notecards, cygwin-sdl, etc. repositories do.
I do not see a pattern between the **have** and the **don't have** repositories.

**Comment by nbriggs** [comment]
We started with one in each of Maiko and Medley, but it was unclear to people reporting a problem with "Interlisp" whether the problem was in the Lisp code or the Maiko code, so we combined them under the Medley Issues.

**Comment by masinter** [comment]
there should only be one repo with issues. If there are others, it's a mistake. 
When the Documentation doesn't match the Implementation, it's open to discussion which should change.
This issue can be closed when I fix clhs etc to not have one.

**Comment by masinter** [comment]
I've removed "Issues" as an option in all repos except medley.
